### PR TITLE
[Flow EVM] Updating chain IDs

### PR DESCRIPTION
**Pull request by ramtinms**
This PR updates the chain IDs, given some recently deployed networks are using the old ones.

### CODE REVIEW

**Comment by ramtinms** [comment]
@m-Peter for visibility, somehow I can't add you as a reviewer.

File: fvm/evm/emulator/config.go [hunk 1]
```diff
@@ -13,8 +13,8 @@ import (
 )
 
 var (
-	FlowEVMTestnetChainID = big.NewInt(666)
-	FlowEVMMainnetChainID = big.NewInt(777)
+	FlowEVMTestnetChainID = big.NewInt(6666)
```
Comment (franklywatson) [inline fvm/evm/emulator/config.go:16]: Already taken: https://github.com/onflow/evm-chains/blob/master/_data/chains/eip155-6666.json

File: fvm/evm/emulator/config.go [hunk 2]
```diff
@@ -13,8 +13,8 @@ import (
...
+	FlowEVMMainnetChainID = big.NewInt(7777)
```
Comment (franklywatson) [inline fvm/evm/emulator/config.go:17]: Likewise: https://github.com/onflow/evm-chains/blob/master/_data/chains/eip155-7777.json

File: fvm/evm/emulator/config.go (hunk 1 above)
Comment (ramtinms) [reply fvm/evm/emulator/config.go:16]: hah it was not showing up on the website, any suggested value ?

Comment (franklywatson) [reply fvm/evm/emulator/config.go:16]: 646 and 747 are available

**Review by franklywatson — APPROVED** [review]

**Comment by m-Peter** [comment]
@ramtinms Looks good :+1:
The EVM Gateway makes use of these 2 constants, so it will automatically pick these values up when we update the version its `flow-go` dependency.

**Review by janezpodhostnik — APPROVED** [review]

**Pull request merged**